src/Uno.UI/UI/Xaml/Media/Animation/Timeline.animation.cs

```diff
@@ -262,7 +262,6 @@ private void OnAnimatorAnimationEnd(object sender, EventArgs e)
 				}
 
 				OnEnd();
-				_startingValue = null;
```

Comment:
Maybe a similar change will be needed for `ColorAnimationUsingKeyFrames` and `DoubleAnimationUsingKeyFrames`

Reply:
true